owncloudApp/src/main/java/com/owncloud/android/ui/fragment/FileDetailFragment.java

```diff
@@ -268,6 +321,12 @@ public void onPrepareOptionsMenu(Menu menu) {
             item.setVisible(false);
             item.setEnabled(false);
         }
+
+        item = menu.findItem(R.id.action_open_in_web);
```

Comment:
I know it was there before but, why sharing a same variable for every menu item you work with? I think it worsens the understandability of the code. Anyway, just a comment, it's fine for me 👍